.github/actions/multi-approvers/__tests__/main.test.js

```diff
@@ -0,0 +1,202 @@
+/**
```

Comment:
What is `__test__` directory?

Reply:
It's the default test directory that jest uses.